### ISSUE TITLE
Any plans to deploy this as a docker container?

### ISSUE THREAD

**Issue by hdelatorre** [ethstakersclub/ethstakersclub#18]

**Comment by ethstakersclub** [comment]
Yes, @rocknet is actively developing an implementation within his repository, which can be found at https://github.com/rocknet/ethiab.

We are open to incorporating his Docker implementation into our project. Please feel free to share any specific requirements or features you'd like us to consider during this integration process.

**Comment by rocknet** [comment]
> Yes, I think @rocknet is actively developing an implementation within his repository, which can be found at https://github.com/rocknet/ethiab.

That's right, I have some early work done on this in the ethstakersclub branch.  It should at least give some ideas, but right now it's coupled to Lodestar, which isn't actually a good archive node.  Plus, it's for dev environments since it doesn't use gunicorn and the security aspect is an afterthought.

I will actually be working on one this week for a prod environment, as I try to get an explorer up for Holesky.

**Comment by hdelatorre** [comment]
> We are open to incorporating his Docker implementation into our project. Please feel free to share any specific requirements or features you'd like us to consider during this integration process.

@rocknet mentioned working on this this weekend for a prod deployment, I'll take a look next week and provide any requirements features I'd be looking for.  Ultimately, I'd like to run this in a Kubernetes environment.
